libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp

```diff
@@ -368,9 +394,13 @@ bool AP_InertialSensor_Invensensev3::hardware_init(void)
         _clip_limit = 29.5f * GRAVITY_MSS;
         break;
     case Invensensev3_Type::ICM42688:
+    case Invensensev3_Type::IIM42652:
     case Invensensev3_Type::ICM42605:
         _clip_limit = 15.5f * GRAVITY_MSS;
         break;
+    case Invensensev3_Type::ICM40605:
+        _clip_limit = 15.5f * GRAVITY_MSS;
+        break;
```

Comment:
This makes it easier to read and is functionally equivalent

```suggestion
    case Invensensev3_Type::IIM42652:
    case Invensensev3_Type::ICM42605:
    case Invensensev3_Type::ICM40605:
        _clip_limit = 15.5f * GRAVITY_MSS;
        break;
```